### DISCUSSION TITLE
makeStyles from MUI

### DISCUSSION THREAD

**Discussion by g0dzcsgo** [swordev/suid#227] [Ideas]
Would it be possible to add makeStyles aswell from MUI? My app heavily relies on it and I cannot use solid if I don't have the ability to makeStyles like I do with MUI. Is there any plans to port this over from MUI?

In react you just import: import { makeStyles } from '@mui/styles';

**Comment by wakaztahir** [comment]
You can do this

```
import styled from "@suid/system/styled"

const MyButton = styled("button")({
  backgroundColor:"red"
})
```
